spring-app/src/main/java/pl/edu/agh/cqm/controller/MonitorController.java

```diff
@@ -0,0 +1,31 @@
+package pl.edu.agh.cqm.controller;
+
+import lombok.AllArgsConstructor;
+import org.springframework.web.bind.annotation.*;
+import pl.edu.agh.cqm.data.dto.MonitorsDTO;
+import pl.edu.agh.cqm.data.dto.MonitorsResponseDTO;
+
+import java.util.List;
+
+@CrossOrigin
+@RestController
+@RequestMapping("api/monitors")
+@AllArgsConstructor
+public class MonitorController {
+
+//    @PutMapping
+//    public void put(
+//            @Valid @RequestBody MonitorsDTO monitorsDTO
+//    ) {
+//    }
+
+    @GetMapping
+    public MonitorsResponseDTO get() {
+        return MonitorsResponseDTO.builder()
+                .monitors(List.of(
+                        MonitorsDTO.builder().id(1).address("1.1.1.1").build(),
+                        MonitorsDTO.builder().id(2).address("2.2.2.2").build()
+                ))
+                .build();
+    }
```

Comment:
commited by mistake?

Reply:
Allows to assure good communication with backend and don't throw errors that something is not implemented